entrypoint.sh

```diff
@@ -25,3 +26,5 @@ fi
 unset JAVA_HOME
 
 sonar-scanner -Dsonar.projectBaseDir=${INPUT_PROJECTBASEDIR} ${INPUT_ARGS}
+
+rm -rf "${INPUT_PROJECTBASEDIR}/.scannerwork"
```

Comment:
Actually, we cannot remove this 🤦 . This would break SonarSource/sonarqube-quality-gate-action (see [this line](https://github.com/SonarSource/sonarqube-quality-gate-action/blob/master/action.yml#L16)).

If the permissions here are an issue, we need to find another way to deal with it.

Reply:
i changed the logic to simply fix the filesystem permissions for this folder. should not cause another issue (hopefully)